Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.py

```diff
@@ -0,0 +1,611 @@
+                outputs=response
+            )
+    else:
+        msg = error_response(err_msg)
+        raise DemistoException(msg)
+
+
+def get_verdict_cmd(client: Client, args: Dict[str, str]) -> CommandResults:
+    taskUuid = args.get(SNXResponse.SNX_TASK_UUID_KEY) or None
+    if taskUuid is None:
+        raise DemistoException("Task UUID Parameter value is not found")
+    else:
+        urlSuffix = "/get_verdict"
+        params = {SNXResponse.SNX_TASK_UUID_KEY: taskUuid}
+        response, err_msg = client.get_response(urlSuffix, params)
+        if response:
+            entryContent = {}
+            finalJson = {}
+            if SNXResponse.SNX_SUCCESS_KEY in response.keys() and SNXResponse.SNX_RESULT_KEY in response.keys():
+                finalJson = response[SNXResponse.SNX_RESULT_KEY]
+                entryContent = {
+                    "SnXAnalysis.Verdict(val.task_uuid == obj.task_uuid)": finalJson
+                }
+            else:
+                finalJson = response
+                entryContent = response
+            readableOutput = tableToMarkdown(f"SecneurX Analysis - Verdict Result: {taskUuid}", t=finalJson)
+            return CommandResults(
+                readable_output=readableOutput,
+                outputs=entryContent,
+                outputs_prefix="",
+                raw_response=finalJson
+            )
+        else:
+            msg = error_response(err_msg)
+            raise DemistoException(msg)
+
+
+def get_completed_cmd(client: Client, args: Dict[str, str]) -> CommandResults:
+    urlSuffix = "/get_completed"
+    params = create_request_json(args)
+    response, err_msg = client.get_response(urlSuffix, params)
+    if response:
+        if SNXResponse.SNX_SUCCESS_KEY in response.keys() and response[SNXResponse.SNX_SUCCESS_KEY] == SNXResponse.SUCCESS:
+            reportList = response[SNXResponse.SNX_RESULT_KEY]
+            if reportList != SNXResponse.SNX_NULL_KEY and len(reportList) > 0:
+                readableOutput = tableToMarkdown("SecneurX Analysis - List of Completed Samples:", t=reportList,
+                                                 headers=[
+                                                     SNXResponse.SNX_TASK_UUID_KEY, SNXResponse.SNX_VERDICT_KEY,
+                                                     SNXResponse.SNX_STATUS_KEY, SNXResponse.SNX_REPORT_KEY])
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs={},
+                    raw_response=reportList
+                )
+            else:
+                msgJson = {"msg": "No samples to display"}
+                readableOutput = tableToMarkdown("SecneurX Analysis - List of Completed Samples: ", msgJson)
+                return CommandResults(
+                    readable_output=readableOutput,
+                    outputs_prefix="",
+                    outputs=msgJson
+                )
```

Comment:
This outputs will not be under a special key.
Consider removing outputs so it doesn't return to context, or add to `output_prefix` like this:
```suggestion
                return CommandResults(
                    readable_output=readableOutput,
                    outputs_prefix="SnXAnalysis.Completed",
                    outputs=msgJson
                )
```